src/windows.rs

```diff
@@ -403,6 +403,14 @@ pub struct ReadDirectoryChangesWatcher {
 }
 
 impl ReadDirectoryChangesWatcher {
+    pub fn new<F: EventFn>(event_fn: F) -> Result<Self> {
+        // create dummy channel for meta event
+        // TODO: determine the original purpose of this - can we remove it?
```

Comment:
I think this is from the original implementation for debouncing where each Watcher also implemented something like "new debounced" (though it was swapped, new was debounced) and setup the channel system internally.

Problem is that we have to pass-through events from the watcher to the debouncer to the user but the user received a debouncer object that has to pass configuration changes back to the watcher. (Which is why I've currently halted adding a new debouncer until I've got a better API idea.)

Reply:
And the same happens but with a raw watcher object without any debouncer, which makes this a really bad data chain.

Reply:
note I just copied this comment over from the old `ReadDirectoryChangesWatcher::new_immediate`, so I don't have context why this comment was added :)